main.py

```diff
@@ -25,7 +25,7 @@ def build_game_controller(width, height, level_path='levels/example.txt') -> Bel
     view = ConsoleView(model, model_vars, writer)
     return BellhopController(model, view)
 
-def build_visuals_controller(width, height, level_path='levels/example.txt') -> BellhopController:
+def build_visuals_controller(width, height, level_path) -> BellhopController:
```

Comment:
whoops these args in the functions of main should have types

Reply:
done